docs/views/examples/pass-data/index.html

```diff
@@ -140,7 +140,7 @@ <h3 class="govuk-heading-s">
   name: "vehicle1[vehicle-features]"
```

Comment:
Thanks @philswan – great spot as there is a comma missing from the example, but it's actually missing from a couple of lines higher, where we set the name:

```suggestion
  name: "vehicle1[vehicle-features]",
```

If you make that change in your prototype, can you confirm that it fixes the error that you were seeing? If so, I can update this PR for you so that it changes the correct line, and we can get this merged.

Reply:
@36degrees that’s done the trick – thanks!